docs/README.md

```diff
@@ -38,6 +41,31 @@ Each page within a Syna based website is created using fragments.
 
 <!-- /TOC -->
 
+## Installation
+
+### Adding Syna to your site
+
+You can use Syna by adding it as a submodule to your website repository and pointing the submodule to the latest release.
+This way whenever you want to update the theme you can just pull the updates and checkout to the latest tag.
+
+```bash
+git submodule init # If you haven't initialized before
+git submodule add git@github.com:okkur/syna.git themes/syna
```

Comment:
https vs git based url?

Reply:
As this is the developer centric option, we might get contributions so keeping git, might be beneficial here. Not sure so.

Reply:
Actually I wrote this because I had totally forgotten about the https url. git@ is just so much easier.